js/models/messages.js

```diff
@@ -1053,8 +1054,28 @@
         //   ? storage.get('profileKey')
         //   : null;
 
+        if (conversation.isPublic()) {
+          const openGroup = {
+            server: conversation.get('server'),
+            channel: conversation.get('channelId'),
+            conversationId: conversation.id,
+          };
+          const openGroupParams = {
+            identifier: this.id,
+            body: this.get('body'),
```

Comment:
what about attachments and quotes?

Reply:
added           `const { body, attachments, preview, quote } = await this.uploadData();` and used it